pkgs/tools/X11/find-cursor/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, libX11, libXdamage, libXrender, libXcomposite, libXext, installShellFiles, git }:
+
+stdenv.mkDerivation {
+  version = "1.6";
+  pname = "find-cursor";
+
+  src = fetchFromGitHub {
+    owner = "arp242";
+    repo = "find-cursor";
+    rev = "45e937a7a334373ee26b6e33341dfb79561d2767";
```

You should actually use version 1.6:
```suggestion
    rev = "v${version}";
```

---

Thanks for kindly remind this